src/scenes/home/home.js

```diff
@@ -142,20 +143,21 @@ class Home extends Component {
       [`${styles[this.state.bgImageStyle]}`]: this.state.bgImage,
     });
     return (
-      <div
-        className={classes}
-        style={
-          this.state.bgImage
-            ? { backgroundImage: `url(${this.state.bgImageUrl})` }
-            : {}
-        }
-      >
+      <div>
+        <div
```

Comment:
So, once we get @ksmacleod99 's work in, you'll drop this div and insert the specific `<HeroBanner />` for the home route, and then in the relevant files for the routes that use HeroBanners (`history`, `our_programs`, `team`)